doc/build/orm/extensions/asyncio.rst

```diff
@@ -720,13 +724,18 @@ constructor::
 
     from asyncio import current_task
 
-    from sqlalchemy.ext.asyncio import async_sessionmaker
-    from sqlalchemy.ext.asyncio import async_scoped_session
-    from sqlalchemy.ext.asyncio import AsyncSession
-
-    async_session_factory = async_sessionmaker(some_async_engine, expire_on_commit=False)
```

Comment:
here also, async_sessionmaker() should be used.   sessionmaker() doesn't handle typing for asyncio correctly so the pattern is async_sessionmaker().